source/PowerAdjusterView.mc

```diff
@@ -137,6 +143,18 @@ class DataField extends Ui.SimpleDataField {
 
         if(info.currentPower != null) {
             avgPower = POWER_MULTIPLIER * SLOPE.interpolate(info.currentPower);
+
+            if (PURE_POWER) {
+                var PB = bikePower.getPedalPowerBalance();
+                var TE = bikePower.getTorqueEffectivenessPedalSmoothness();
+                if (PB and TE) {
+                    var Er = TE.rightTorqueEffectiveness;
```

Comment:
For the record:
Stages (expectedly) produce left-only readings for both TE and PB.
I don't know if it's a radio or a software issue but once in 5-20 seconds readings disappear (showing "-") and then reappear again. I guess it'd be safer (for me :-) ) to keep either the last read value or an average and use it if an instant reading is unavailable.